### PR TITLE
Cookie settings confirmation: Added previous link, removed auto redirect

### PR DESCRIPTION
**Pull request by boxadesign**
### What is the context of this PR?
Updated the cookies settings confirmation panel.
- Removed auto redirect if referrer.
- Added link with referrer if one exists
- Added accessibility attributes.
- Added styling changes to panel.

### CODE REVIEW

File: src/js/cookies-settings.js [hunk 1]
```diff
@@ -55,31 +60,33 @@ export default class CookiesSettings {
...
-    }
-  }
-
   showConfirmationMessage() {
-    const confirmationMessage = document.querySelector('.cookies-confirmation-message');
-
-    document.body.scrollTop = document.documentElement.scrollTop = 0;
-    confirmationMessage.classList.remove('u-d-no');
+    this.confirmationMessage.classList.remove('u-d-no');
+    this.confirmationMessage.scrollIntoView();
```
Comment (jrbarnes9) [inline src/js/cookies-settings.js:71]: I think scrolling to top of body was better, as it mimics a page reload and consistent with error validation.  It does look disorientating without the context of the page header in view.

File: src/patterns/cookies-settings/examples/cookie-settings-page/index.njk [hunk 2]
```diff
@@ -71,14 +71,18 @@ height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
...
         {% call onsPanel({
             "classes": "u-mb-s u-d-no cookies-confirmation-message",
-            "type": "success"
+            "type": "success",
+            "icon": "check",
+            "iconSize": "xl"
         }) %}
-            <p class="u-fs-r-b">Your cookie settings have been saved</p>
+            <h2>Your cookie settings have been saved</h2>
             <p>Some parts of the census may use additional cookies and will have their own cookie policy and banner.</p>
```
Comment (jrbarnes9) [inline src/patterns/cookies-settings/examples/cookie-settings-page/index.njk:82]: Can we remove bottom margin on this `p` if no link follows it?

File: src/patterns/cookies-settings/examples/cookie-settings-page/index.njk [hunk 3]
```diff
@@ -71,14 +71,18 @@ height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
 
 {% block main %}
     <div class="page__body">
+        <h1 class="u-fs-xxl">Cookies on census.gov.uk</h1>
         {% call onsPanel({
             "classes": "u-mb-s u-d-no cookies-confirmation-message",
```
Comment (jrbarnes9) [inline src/patterns/cookies-settings/examples/cookie-settings-page/index.njk:76]:
```suggestion
            "classes": "u-mb-m u-d-no cookies-confirmation-message",
```

**Review by jrbarnes9 — APPROVED** [review]

**Review by rmccar — APPROVED** [review]

**Comment by jrbarnes9** [comment]
[Remove automatic re-direct in cookie confirmation js, and replace with a link](https://trello.com/c/yyCHH73A/1043-remove-automatic-re-direct-in-cookie-confirmation-js-and-replace-with-a-link)

**Pull request merged**